packages/docusaurus/src/server/configValidation.ts

```diff
@@ -40,6 +40,7 @@ export const DEFAULT_CONFIG: Pick<
 const PluginSchema = Joi.alternatives().try(
   Joi.string(),
   Joi.array().items(Joi.string().required(), Joi.object().required()).length(2),
+  Joi.bool(), // In case of conditional adding of plugins.
```

Comment:
not sure but I think it's possible to do `Joi.bool().equal(false)`. I think passing true should be rejected

Reply:
Agree, fixed it.